### DISCUSSION TITLE
How can I find the email address of a Github organization?

### DISCUSSION THREAD

**Discussion by sakthivelj** [community/community#79106] [Other Feature Feedback, Questions, & Ideas]
Hello, We have created an organization for our company, but we forgot which email ID we used to create it. How can we find out the email address associated with our organization at Croatia-technologies https://github.com/coratia-technologies? 

I have raised a ticket, but I haven't received any response yet https://support.github.com/ticket/personal/0/2481497. Could you please let me know how long it will take to get a response?

Thanks
Sakthivelj

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @sakthivelj ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by sakthivelj** [reply]
@queenofcorgis Thanks, I updated the category
